src/styles/annotator/highlights.scss

```diff
@@ -1,5 +1,26 @@
-@use '../variables' as var;
-@use '../mixins/a11y';
+$color-highlight: rgba(255, 255, 60, 0.4);
+$color-highlight-second: rgba(206, 206, 60, 0.4);
+$color-highlight-focused: rgba(156, 230, 255, 0.5);
+
+// Hide content from sighted users but make it visible to screen readers.
+//
+// Resources:
+// - https://webaim.org/techniques/css/invisiblecontent/ (see "CSS clip")
+// - https://cloudfour.com/thinks/see-no-evil-hidden-content-and-accessibility/#showing-additional-content-for-screen-readers
+@mixin screen-reader-only {
```

Comment:
It looks to me that this class is almost identical to Tailwind's `sr-only` class. Probably worth noting that here.